openex-front/package.json

```diff
@@ -99,7 +100,8 @@
     "build": "node builder/prod/prod.js",
     "lint": "cross-env DEBUG=eslint:cli-engine TIMING=1 eslint --max-warnings 0 --cache -c .eslintrc.json src",
     "control": "yarn audit --groups dependencies --summary",
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "generate-types-from-api": "swagger-typescript-api -p http://localhost:8080/api-docs -o src/utils -n index.d.ts"
```

Comment:
Weird to enforce a localhost, wouldn't it be better to have it relative ?

Reply:
Do you think we will generate the types on another instance than local ?

Reply:
May be I'm missing something, but for OpenCTI we generate types with drone when building. Won't you be missing something if we stick with localhost ? 
Especially when the package support relative path :
` npx swagger-typescript-api -p ./swagger.json -o ./src -n myApi.ts`

Reply:
Ah ok I see what you mean.
I am not sure if we want to automatically generate types at first however we can have it relative to be future proof

Reply:
_⛔   error while fetching data from URL "./swagger.json" TypeError: Only absolute URLs are supported_

:'(